### DISCUSSION TITLE
Embedded Lyrics

### DISCUSSION THREAD

**Discussion by Satt2ds** [namidaco/namida#426] [General]
I have embedded lyrics on my file. But i can't use. The only way is make a ".LRC", copying text from tags and send to another app to create a LRC.

**Comment by MSOB7YY** [comment]
which version u on? u sure "prioritize embedded lyrics" is enabled? share ur logs and a sample file

**Reply by Satt2ds** [reply]
v4.8.1-beta,i set on lyrics source as local, I was messing around and figured out how to "FIX", i edited tag, modified lyrics tag, and saved, the embedded lyrics appear for some reason, i not sure if thats correct, but it worked. 

And about  "prioritize embedded lyrics", i find and are set as True.
Logs:
[logs_4.8.1-beta_241208024.txt](https://github.com/user-attachments/files/18142072/logs_4.8.1-beta_241208024.txt)

Edit:
I tested indexing the music file on another phone, and it worked properly
But I want to use it on my main phone
Edit: :T I reinstalled the app, and it worked. I tried a backup, but the metadata disappeared again. I must have set something wrong, but I don't know what it was.

**Reply by MSOB7YY** [reply]
maybe ur backup is missing the lyrics in tracks info, haveu tried reindexing after restoring backup?
